src/api.test.js

```diff
@@ -0,0 +1,46 @@
+jest.mock('axios');
+
+import axios from 'axios';
+
+import api from './api';
+import config from './config.json';
+
+import { toOneLineString } from './utils/template-string.util';
+
+describe('#cantidadOfertas', () => {
+  const pais = 'pais';
+  const expectedLaboralLength = 'expectedLaboralLength';
+  const expectedFailureResponse = 0;
+
+  it('should return length of Laboral for success response', async () => {
+    axios.post.mockReturnValue(
+      Promise.resolve({
+        data: { data: { Laboral: { length: expectedLaboralLength } } },
+      })
+    );
+
+    const laboralLength = await api.cantidadOfertas(pais);
+
+    expect(laboralLength).toBe(expectedLaboralLength);
+    expect(axios.post).toBeCalledWith(
+      config.urlGraphQL,
+      expect.any(Object)
+    );
+    const [[, { query }]] = axios.post.mock.calls;
```

Comment:
jest api to get mock arguments doc: https://jestjs.io/docs/en/mock-function-api#mockfnmockcalls